config/env.go

```diff
@@ -20,4 +18,11 @@ const (
 	EnvFortaShardID          = "FORTA_SHARD_ID"
 	EnvFortaShardCount       = "FORTA_SHARD_COUNT"
 	EnvFortaTokenExchangeURL = "FORTA_TOKEN_EXCHANGE_URL"
+
+	EnvJsonRpcHost           = "JSON_RPC_HOST"
+	EnvJsonRpcPort           = "JSON_RPC_PORT"
+	EnvCacheJsonRpcCachePort = "CACHE_JSON_RPC_PORT"
```

Comment:
Can we prefix all of these new cache env vars like `JSON_RPC_CACHE_*`? E.g. `JSON_RPC_CACHE_PORT`, `JSON_RPC_CACHE_REQUEST_TIMEOUT` etc. Prefixing with `CACHE` could be confusing if we introduce another cache for the bots to use.